backend/src/controller/chatController.ts

```diff
@@ -34,7 +34,7 @@ async function handleLowLevelChat(
 		req.session.levelState[currentLevel].sentEmails,
 		currentLevel
 	);
-	chatResponse.reply = openAiReply.completion?.content ?? '';
+	chatResponse.reply = openAiReply.completion?.content?.toString() ?? '';
```

Comment:
How does this behave when `openAiReply.completion.content` is of type `ChatCompletionContentPart[]`? Or is that not expected to happen? Couldn't find much online about it

Reply:
my understanding of this is that this would only be used in cases where we can input images and text - as we can't do that it should only ever by a string :) 

These are the types 

export type ChatCompletionMessageParam =
  | ChatCompletionSystemMessageParam
  | ChatCompletionUserMessageParam
  | ChatCompletionAssistantMessageParam
  | ChatCompletionToolMessageParam
  | ChatCompletionFunctionMessageParam;

all of those ..MessageParams have content: string except this user input ones

export interface ChatCompletionUserMessageParam {
  /**

   * The contents of the user message.
   */
  content: string | Array<ChatCompletionContentPart> | null;

  /**
   * The role of the messages author, in this case `user`.
   */
  role: 'user';
}

which is for image input

Reply:
thank you for explaining this to me. Gosh, this API is complicated!